src/normalizer.ts

```diff
@@ -145,6 +145,22 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+/**
+ * `tsEnumNames` is being replaced by `x-enum-varnames`. Since we don't want to
+ * break existing usage of `tsEnumNames`, we'll convert `tsEnumNames` properties
+ * to `x-enum-varnames`.
+ */
+rules.set('Convert tsEnumNames -> x-enum-varnames', schema => {
+  if (schema.tsEnumNames) {
+    if (Object.keys(schema).includes('x-enum-varnames')) {
+      throw new Error('Cannot set both x-enum-varnames and tsEnumNames in the same schema. Only use x-enum-varnames.')
```

Comment:
Mind being a little more lenient, and only throwing if the two don't match? These should also have tests, if possible.

ie.

```js
// ok
{
  tsEnumNames: ['a']
}

// ok
{
  x-enum-varnames: ['a']
}

// ok
{
  tsEnumNames: ['a'],
  x-enum-varnames: ['a']
}

// error
{
  tsEnumNames: ['a'],
  x-enum-varnames: []
}

// error
{
  tsEnumNames: [],
  x-enum-varnames: ['a']
}

// error
{
  tsEnumNames: ['a'],
  x-enum-varnames: ['b']
}

// error
{
  tsEnumNames: ['a'],
  x-enum-varnames: ['a', 'b']
}
```